draft-quic-atsss-reqs.mkd

```diff
@@ -619,7 +632,34 @@ Both proposals rely upon single QUIC connections and inherit thus the same issue
 
 In 3GPP architectures, a variety of encapsulations are already used to convey user data. The use of QUIC as a tunneling method for ATSSS purposes will thus add an extra overhead. This overhead is even exacerbated when the user data is forwarded over a non-3GPP network access given than an IPsec tunnel is used to graft the UE to the 5G Core Network. For example, the overhead without taking into account the QUIC tunnel is at least 142 octets for an IPv6 packet forwarded over a non-3GPP network access. As a reminder, ATSSSv1 does not add any overhead. 
 
-More investigation is required to assess whether the ATSSSv2 overhead is an issue. Solutions to optimize that overhead might be considered, if needed. 
+More investigation is required to assess whether the ATSSSv2 overhead is an issue. Solutions to optimize that overhead might be considered, if needed.
+
+## Packet order reconstruction for (MP-)QUIC Splitting Mode
+
+Both tunnel solutions in {{sec-quic-tunnel}} and {{sec-mpquic-tunnel}} allow the
+splitting mode for simultaneously sending data over multiple paths. The respective
+receiver side of the tunnel termination points has to deal with the unreliable
+transport character of the Unreliable Datagram Extension for QUIC and a possible
+reconstruction of the original packet ordering. In practice, it is assumed that some sort of
+reconstruction has to be applied. Different to MPTCP, which is reliable by nature
+and has just to wait for missing segments, such reconstruction mechanisms have to
+account that packet loss may occur for any of the paths within the QUIC tunnel.
+Another issue is the packet reordering introduced when a QUIC tunnel
+communication is split across paths exposing very distinct latencies. In theory,
+applications have to deal with packet reordering, since for example typical best
+effort Internet traffic has no mechanism to prevent it.  However, in practice,
+without any multipath extension, packet reordering can be assumed to be very
+limited. Therefore most piggybacked services on top of a Unreliable Datagrame
```

Comment:
Datagrame --> Datagram

Reply:
Thx Anna.

Solved in https://github.com/obonaventure/draft-quic-atsss-reqs/pull/88/commits/8dd31945a8d5664634cc4118d4c1c6741a65885c